pkg/smbios/table.go

```diff
@@ -36,11 +36,49 @@ const (
 	outOfSpec = "<OUT OF SPEC>"
 )
 
+// ModifyStrings modifies the strings part of the table
+func (t *Table) ModifyStrings(newStr []string) {
+	t.strings = newStr
+}
+
+// ModifyData modifies the data part of the table
+func (t *Table) ModifyData(newData []byte) {
+	t.data = newData
+}
+
 // Len returns length of the structured part of the table.
 func (t *Table) Len() int {
 	return len(t.data)
 }
 
+// TotalLen returns the total size of the table in bytes as presented in memory
+func (t *Table) TotalLen() int {
```

Comment:
This is the same as len(t.GetRaw()) right? Why does it need to exist?

Reply:
Yes, but I prefer to add it since it'll be used very often and to make difference to `Len()`

Reply:
Note that getRaw are old bytes read from memory.

TotalLen() could reflect new total len, if we added a few new strings.

Perhaps update the function comment ?